src/azure-cli-core/azure/cli/core/azclierror.py

```diff
@@ -62,6 +62,11 @@ def set_exception_trace(self, exception_trace):
     def print_error(self):
         from azure.cli.core.azlogging import CommandLoggerContext
         from azure.cli.core.style import print_styled_text
+
```

Comment:
Is there any case that the error is not type of AzCLIError while we still want to print trace?

Reply:
Unless intercepted, other types are handled by `azure.cli.core.util.handle_exception`:

https://github.com/Azure/azure-cli/blob/325dfa64d8b77bfddb3b087162ad98acee5037ab/src/azure-cli-core/azure/cli/core/util.py#L56

If `azure.cli.core.util.handle_exception` still can't handle it, it will be handled by

https://github.com/Azure/azure-cli/blob/325dfa64d8b77bfddb3b087162ad98acee5037ab/src/azure-cli-core/azure/cli/core/util.py#L167-L172

so the trace is always printed for non-`AzCLIError` errors.